### DISCUSSION TITLE
Would update process impact the query?

### DISCUSSION THREAD

**Discussion by yikakabu** [mediagis/nominatim-docker#357] [Q&A]
Hello, I wonder if the updating will impact the query of server.
And How often would you suggest update the datebase? Is that controled by the env REPLICATION_UPDATE_INTERVAL?
Please forgive my naive question. Thanks in Advance.

**Comment by mtmail** [comment]
There is a small impact because updating consumes CPU and writing to the database tables makes the database cache less efficient. But I wouldn't worry on a multi-CPU system.

https://nominatim.openstreetmap.org/ updates every minute from https://planet.osm.org/

When you use an extract, e.g. Geofabrik is mentioned on https://github.com/mediagis/nominatim-docker/tree/master/4.0#openstreetmap-data-extracts then you have to check how often they publish diff/change files. Geofabrik for example only once per day. https://wiki.openstreetmap.org/wiki/Planet.osm#Planet.osm_mirrors has a list of sources.

**Comment by yikakabu** [comment]
Thanks for your reply.
And I want to know how much storage  the volumes need respectively for /var/lib/postgresql/12/main and /nominatim/flatnode in the situation of planet data.
According to the Nominatim doc, full planet install will need at least 900G of disk space. But I need to decide how to allocate the 900G to /var/lib/postgresql/12/main and /nominatim/flatnode.

**Comment by mtmail** [comment]
OpenStreetMap data is growing every day so even the 900GB might no longer work. https://wiki.openstreetmap.org/wiki/Stats#Nodes,_ways_and_relations Plan 10% extra: 900GB for Postgresql plus 100GB for flatnodefile.

**Comment by yikakabu** [comment]
Thanks a lot.

**Comment by yikakabu** [comment]
I set the env REPLICATION_URL to https://ftpmirror.your.org/pub/openstreetmap/replication/day/ and leave the other REPLICATION env default. But I can not see any log about the update info for a week after the import, the last log is still `LOG: database system is ready to accept connections`. 
Dose the update work and how can I inspect the update process?

**Comment by mtmail** [comment]
I think you're looking at the log of the database, not Nominatim. Check the contents of the postgresql `import_status` table, it should show a recent date.

**Comment by philipkozeny** [comment]
Did you start the replication process manually, like it is described here: https://github.com/mediagis/nominatim-docker/tree/master/4.0#updating-the-database ? This does not happen automatically.
